package.json

```diff
@@ -24,7 +24,13 @@
     "prepare-release": "./prepare-release.sh",
     "release": "./release.sh",
     "prune-arc-build": "node ./pruneArcBuild.js",
-    "postinstall": "npm run prune-arc-build"
+    "postinstall": "npm run prune-arc-build",
+    "verify.initialize": "npm explore @daostack/arc -- npm install openzeppelin-solidity @daostack/infra --no-save && npm explore @daostack/infra -- npm install openzeppelin-solidity --no-save && npm explore @daostack/arc-hive -- npm install openzeppelin-solidity --no-save",
+    "verify.build": "node node_modules/typescript/bin/tsc verify.ts -sourceMap",
```

Comment:
there's `ts-node` that you could use that executes typescript directly.

Reply:
what is there is sufficient without adding another tool

Reply:
but it saves you the mental headache of remembering to do the build step.

Reply:
verify.initialize and verify.build are already run in prepare-release, prior to running verify.